custom/_legacy/hsph/tests.py

```diff
@@ -39,14 +39,18 @@ def test_chunk_size(self):
             domain = call_args[0][1]
             case_ids = [get_case_id(case_block) for case_block in case_blocks]
             call_args_list.append((case_ids, domain))
+
         expected = [
```

Comment:
These tests are not super necessary now, but keeping it in case we need to revert back